docs/mastering-plone-5/behaviors_1.md

```diff
@@ -198,6 +198,6 @@ We must add the behavior to {file}`profiles/default/types/talk.xml`:
 </object>
 ```
 
-[plone5_fieldset]: https://5.docs.plone.org/develop/addons/schema-driven-forms/customising-form-behaviour/fieldsets.html?highlight=fieldset
+[plone5_fieldset]: https://5.docs.plone.org/develop/addons/schema-driven-forms/customizing-form-behaviour/fieldsets.html?highlight=fieldset
```

Comment:
Do not "fix" misspellings in URLs because that will break links. Nonetheless, the GET parameter should be removed, as it causes a redirect.

```suggestion
[plone5_fieldset]: https://5.docs.plone.org/develop/addons/schema-driven-forms/customising-form-behaviour/fieldsets.html
```

Reply:
Also make sure to run `make linkcheckbroken` to check for broken links. That test would have caught the mistake for you.